docs/pages/api-reference/operators/window.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Window
+order: 0
+status: published
+---
+### Window
+
+<Divider>
+<LeftSection>
+Operator to do aggregation of events into windows based on their timestamps. Window operator must always be preceded by a [groupby](/api-reference/operators#groupby) operator.
+
+#### Parameters
+
+<Expandable title="type" type="Window">
+
+The type of window to group event on. Possible values are `"hopping"`, `"tumbling"`, `"session"`.
+
+`"hopping"`: Take in two parameters `"duration"` and `"stride"`, the `"duration"` parameters indicate how long the window is while `"gap"` indicate how far apart the window should be. Hopping window can overlap meaning one event can belong to two different windows. `"stride"` should be smaller than `"duration"`.
+
+`"tumbling"`: Take in one parameter `"duration"` indicate how long the window is. The tumbling windows do not overlapped and cover the whole range. 
+
+`"session`: Take in one parameter `"gap"` indicating how close together two events are to be group into one window. Session window do not overlapped and always at least `"gap"` apart.
+</Expandable>
+
+<Expandable title="gap" type="Duration">
+
+The parameters for `"session"` windows. Indicating how far apart the event should be to in one window.  Possible values are any [time duration](/api-reference/data-types/duration)
+</Expandable>
+
+<Expandable title="duration" type="Duration">
+
+The parameters for `"tumbling"` or `"hopping"` windows. Indicating the length of each window. Possible values are any [time duration](/api-reference/data-types/duration)
+</Expandable>
+
+<Expandable title="stride" type="Duration">
+
+The parameters for `"hopping"` windows. Indicating how far the windows from each other. Possible values are any [time duration](/api-reference/data-types/duration)
+</Expandable>
+
+<Expandable title="field" type="str">
+The name of the field in the output dataset that should store the result of this aggregation. This field is expected to be of type `"Window"`.
+</Expandable>
+
+#### Returns
```

Comment:
Can you plz document the `Errors` section as well - what are most important commit type or runtime errors?